### PR TITLE
Relicense as Apache-2.0.

### PR DESCRIPTION
**Pull request by jeandudey**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,8 +5,16 @@ protocol as the serialization format for messages.
 
 # License
 
-<sup>Copyright 2018 Jean Pierre Dudey <jeandudey@hotmail.com></sup>
+<sup>Copyright 2020 Jean Pierre Dudey <me@jeandudey.tech></sup>
```
Comment (vorot93) [inline README.md]:
```suggestion
<sup>Copyright 2018-2020 Jean Pierre Dudey <me@jeandudey.tech></sup>
<sup>Copyright 2020 Artem Vorotnikov <artem@vorotnikov.me></sup>
```

File: src/de.rs [hunk 2]
```diff
@@ -1,10 +1,16 @@
-// Copyright 2018 Jean Pierre Dudey <jeandudey@hotmail.com>
+// Copyright 2020 Jean Pierre Dudey <me@jeandudey.tech>
```
Comment (vorot93) [inline src/de.rs]:
```suggestion
// Copyright 2018-2020 Jean Pierre Dudey <me@jeandudey.tech>
// Copyright 2020 Artem Vorotnikov <artem@vorotnikov.me>
```

File: src/header.rs [hunk 3]
```diff
@@ -1,10 +1,16 @@
-// Copyright 2018 Jean Pierre Dudey <jeandudey@hotmail.com>
+// Copyright 2020 Jean Pierre Dudey <me@jeandudey.tech>
```
Comment (vorot93) [inline src/header.rs]:
```suggestion
// Copyright 2018-2020 Jean Pierre Dudey <me@jeandudey.tech>
// Copyright 2020 Artem Vorotnikov <artem@vorotnikov.me>
```

File: src/lib.rs [hunk 4]
```diff
@@ -1,10 +1,16 @@
-// Copyright 2018 Jean Pierre Dudey <jeandudey@hotmail.com>
+// Copyright 2020 Jean Pierre Dudey <me@jeandudey.tech>
```
Comment (vorot93) [inline src/lib.rs]:
```suggestion
// Copyright 2018-2020 Jean Pierre Dudey <me@jeandudey.tech>
// Copyright 2020 Artem Vorotnikov <artem@vorotnikov.me>
```

File: src/raw_size.rs [hunk 5]
```diff
@@ -1,10 +1,16 @@
-// Copyright 2018 Jean Pierre Dudey <jeandudey@hotmail.com>
+// Copyright 2020 Jean Pierre Dudey <me@jeandudey.tech>
```
Comment (vorot93) [inline src/raw_size.rs]:
```suggestion
// Copyright 2018-2020 Jean Pierre Dudey <me@jeandudey.tech>
// Copyright 2020 Artem Vorotnikov <artem@vorotnikov.me>
```

File: src/ser.rs [hunk 6]
```diff
@@ -1,10 +1,16 @@
-// Copyright 2018 Jean Pierre Dudey <jeandudey@hotmail.com>
+// Copyright 2020 Jean Pierre Dudey <me@jeandudey.tech>
```
Comment (vorot93) [inline src/ser.rs]:
```suggestion
// Copyright 2018-2020 Jean Pierre Dudey <me@jeandudey.tech>
// Copyright 2020 Artem Vorotnikov <artem@vorotnikov.me>
```

File: utils/src/blob.rs [hunk 7]
```diff
@@ -1,10 +1,16 @@
-// Copyright 2018 Jean Pierre Dudey <jeandudey@hotmail.com>
+// Copyright 2020 Jean Pierre Dudey <me@jeandudey.tech>
```
Comment (vorot93) [inline utils/src/blob.rs]:
```suggestion
// Copyright 2018-2020 Jean Pierre Dudey <me@jeandudey.tech>
// Copyright 2020 Artem Vorotnikov <artem@vorotnikov.me>
```

File: utils/src/bytes_uuid.rs [hunk 8]
```diff
@@ -1,10 +1,16 @@
-// Copyright 2018 Jean Pierre Dudey <jeandudey@hotmail.com>
+// Copyright 2020 Jean Pierre Dudey <me@jeandudey.tech>
```
Comment (vorot93) [inline utils/src/bytes_uuid.rs]:
```suggestion
// Copyright 2018-2020 Jean Pierre Dudey <me@jeandudey.tech>
// Copyright 2020 Artem Vorotnikov <artem@vorotnikov.me>
```

File: utils/src/lib.rs [hunk 9]
```diff
@@ -1,10 +1,16 @@
-// Copyright 2018 Jean Pierre Dudey <jeandudey@hotmail.com>
+// Copyright 2020 Jean Pierre Dudey <me@jeandudey.tech>
```
Comment (vorot93) [inline utils/src/lib.rs]:
```suggestion
// Copyright 2018-2020 Jean Pierre Dudey <me@jeandudey.tech>
// Copyright 2020 Artem Vorotnikov <artem@vorotnikov.me>
```

**Review by vorot93 — CHANGES_REQUESTED** [review]

**Comment by jeandudey** [comment]
Changes applied

**Review by vorot93 — APPROVED** [review]

**Pull request merged**